### DISCUSSION TITLE
Repo-based achievements

### DISCUSSION THREAD

**Discussion by mkArtakMSFT** [community/community#75224] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be great to have an ability to define achievements per organization and/or repository.
This will help with growing communities a lot, as it will allow recognition based on unique contributions. And it will be easy to understand a user's impact by just looking at their achievements.

Here is a specific example, where I need this. I'm a member of the ASP.NET Core team at Microsoft, and I'd like to grant badges to our valued contributors, based on what they actualy did and how much.

I assume there is already similar feedback, but I couldn't find it. Hopefully this will get enough attention to turn it into a feature request.

**Comment by ghassensouissi** [comment]


**Reply by mkArtakMSFT** [reply]
Thanks @ghassensouissi.
Can you point me to a documentation and/or guidance about how to submit an official proposal?

**Reply by mkArtakMSFT** [reply]
@ghassensouissi which repo should I post the proposal in? Sorry, it's just not clear from this response.

**Reply by ghassensouissi** [reply]
No need to apologize! I'm Happy to help 

Well to identify the correct repository for your proposal, check the project's documentation or community guidelines. 

Look for sections related to contributions, proposals, or discussions.

**Comment by ahpook** [comment]
Hi @mkArtakMSFT - thanks for the feedback! There is an existing discussion thread requesting this enhancement here: https://github.com/orgs/community/discussions/19861 

We are also tracking this internally. I don't have a roadmap commitment from the product team but have added your request and use-case to the issue.

**Comment by ettaboyle** [comment]
+1 to @ahpook's comment - thank you for sharing that information! 

@ghassensouissi it looks like the response you shared caused confusion, and I suspect it may have been created with generative AI. We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  

You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by ghost** [comment]
Yeah this idea is awesome. I think this would help the `Repository`or an `Organization` in gaining more _collaborators_ and more _contributions_.
